src/sketchup_objects/link_entities/spring_cylinder.rb

```diff
@@ -2,10 +2,12 @@
 class SpringCylinder < SketchupObject
   attr_reader :material
 
-  def initialize(parent, length, diameter, id = nil)
+  CYLINDER_RADIUS = 0.5
+
+  def initialize(parent, length, spring_length, spring_diameter, id = nil)
```

Comment:
Let's do id as an named (optional) parameter, if it does not break other existing calls ```initialize(...,id: nil)```

Reply:
renamed in de186d8